### ISSUE TITLE
Monaco editor does not load

### ISSUE THREAD

**Issue by eXistenZe** [dnnsoftware/Dnn.Platform#5347]
## Description of bug
Opening the config manager does not load the editor.

## Steps to reproduce
Open config manager in persona bar.

## Current behavior
loader.js is in fact missing from the folder.
editor.main.js is present

```
ConfigConsole.js?cdv=83:237          GET https://x.app/pt-pt/Resources/Shared/components/MonacoEditor/loader.js net::ERR_ABORTED 404
initConfigConsole @ ConfigConsole.js?cdv=83:237
init @ ConfigConsole.js?cdv=83:196
(anonymous) @ util.js?cdv=83:30
execCb @ require.js?cdv=83:1696
check @ require.js?cdv=83:878
(anonymous) @ require.js?cdv=83:1139
(anonymous) @ require.js?cdv=83:134
(anonymous) @ require.js?cdv=83:1189
each @ require.js?cdv=83:59
emit @ require.js?cdv=83:1188
check @ require.js?cdv=83:938
(anonymous) @ require.js?cdv=83:1139
(anonymous) @ require.js?cdv=83:134
(anonymous) @ require.js?cdv=83:1189
each @ require.js?cdv=83:59
emit @ require.js?cdv=83:1188
check @ require.js?cdv=83:938
enable @ require.js?cdv=83:1176
init @ require.js?cdv=83:788
callGetModule @ require.js?cdv=83:1203
completeLoad @ require.js?cdv=83:1590
onScriptLoad @ require.js?cdv=83:1717
load (async)
req.load @ require.js?cdv=83:1942
load @ require.js?cdv=83:1685
load @ require.js?cdv=83:834
fetch @ require.js?cdv=83:824
check @ require.js?cdv=83:856
enable @ require.js?cdv=83:1176
enable @ require.js?cdv=83:1557
(anonymous) @ require.js?cdv=83:1161
(anonymous) @ require.js?cdv=83:134
each @ require.js?cdv=83:59
enable @ require.js?cdv=83:1113
init @ require.js?cdv=83:788
callGetModule @ require.js?cdv=83:1203
completeLoad @ require.js?cdv=83:1590
onScriptLoad @ require.js?cdv=83:1717
load (async)
req.load @ require.js?cdv=83:1942
load @ require.js?cdv=83:1685
load @ require.js?cdv=83:834
fetch @ require.js?cdv=83:824
check @ require.js?cdv=83:856
enable @ require.js?cdv=83:1176
enable @ require.js?cdv=83:1557
(anonymous) @ require.js?cdv=83:1161
(anonymous) @ require.js?cdv=83:134
each @ require.js?cdv=83:59
enable @ require.js?cdv=83:1113
init @ require.js?cdv=83:788
(anonymous) @ require.js?cdv=83:1460
setTimeout (async)
req.nextTick @ require.js?cdv=83:1815
localRequire @ require.js?cdv=83:1449
requirejs @ require.js?cdv=83:1797
loadTempl @ util.js?cdv=83:21
loadTemplate @ util.js?cdv=83:64
(anonymous) @ main.js?cdv=83:294
r.complete @ jquery.js?cdv=83:2
c @ jquery.js?cdv=83:2
fireWith @ jquery.js?cdv=83:2
u @ jquery.js?cdv=83:2
S.fx.tick @ jquery.js?cdv=83:2
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
S.fx.start @ jquery.js?cdv=83:2
S.fx.timer @ jquery.js?cdv=83:2
ft @ jquery.js?cdv=83:2
a @ jquery.js?cdv=83:2
dequeue @ jquery.js?cdv=83:2
(anonymous) @ jquery.js?cdv=83:2
setTimeout (async)
(anonymous) @ jquery.js?cdv=83:2
dequeue @ jquery.js?cdv=83:2
(anonymous) @ jquery.js?cdv=83:2
each @ jquery.js?cdv=83:2
each @ jquery.js?cdv=83:2
queue @ jquery.js?cdv=83:2
S.fn.delay @ jquery.js?cdv=83:2
(anonymous) @ main.js?cdv=83:291
r.complete @ jquery.js?cdv=83:2
c @ jquery.js?cdv=83:2
fireWith @ jquery.js?cdv=83:2
u @ jquery.js?cdv=83:2
S.fx.tick @ jquery.js?cdv=83:2
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
S.fx.start @ jquery.js?cdv=83:2
S.fx.timer @ jquery.js?cdv=83:2
ft @ jquery.js?cdv=83:2
a @ jquery.js?cdv=83:2
dequeue @ jquery.js?cdv=83:2
(anonymous) @ jquery.js?cdv=83:2
each @ jquery.js?cdv=83:2
each @ jquery.js?cdv=83:2
queue @ jquery.js?cdv=83:2
animate @ jquery.js?cdv=83:2
handleLoadPanel @ main.js?cdv=83:288
handleClickOnHoverMenuItem @ main.js?cdv=83:918
dispatch @ jquery.js?cdv=83:2
v.handle @ jquery.js?cdv=83:2
require.js?cdv=83:1961          GET https://c.app/pt-pt/Resources/Shared/components/MonacoEditor/editor/editor.main.js?cdv=83 net::ERR_ABORTED 404
req.load @ require.js?cdv=83:1961
load @ require.js?cdv=83:1685
load @ require.js?cdv=83:834
fetch @ require.js?cdv=83:824
check @ require.js?cdv=83:856
enable @ require.js?cdv=83:1176
enable @ require.js?cdv=83:1557
(anonymous) @ require.js?cdv=83:1161
(anonymous) @ require.js?cdv=83:134
each @ require.js?cdv=83:59
enable @ require.js?cdv=83:1113
init @ require.js?cdv=83:788
(anonymous) @ require.js?cdv=83:1460
setTimeout (async)
req.nextTick @ require.js?cdv=83:1815
localRequire @ require.js?cdv=83:1449
requirejs @ require.js?cdv=83:1797
initConfigConsole @ ConfigConsole.js?cdv=83:240
init @ ConfigConsole.js?cdv=83:196
(anonymous) @ util.js?cdv=83:30
execCb @ require.js?cdv=83:1696
check @ require.js?cdv=83:878
(anonymous) @ require.js?cdv=83:1139
(anonymous) @ require.js?cdv=83:134
(anonymous) @ require.js?cdv=83:1189
each @ require.js?cdv=83:59
emit @ require.js?cdv=83:1188
check @ require.js?cdv=83:938
(anonymous) @ require.js?cdv=83:1139
(anonymous) @ require.js?cdv=83:134
(anonymous) @ require.js?cdv=83:1189
each @ require.js?cdv=83:59
emit @ require.js?cdv=83:1188
check @ require.js?cdv=83:938
enable @ require.js?cdv=83:1176
init @ require.js?cdv=83:788
callGetModule @ require.js?cdv=83:1203
completeLoad @ require.js?cdv=83:1590
onScriptLoad @ require.js?cdv=83:1717
load (async)
req.load @ require.js?cdv=83:1942
load @ require.js?cdv=83:1685
load @ require.js?cdv=83:834
fetch @ require.js?cdv=83:824
check @ require.js?cdv=83:856
enable @ require.js?cdv=83:1176
enable @ require.js?cdv=83:1557
(anonymous) @ require.js?cdv=83:1161
(anonymous) @ require.js?cdv=83:134
each @ require.js?cdv=83:59
enable @ require.js?cdv=83:1113
init @ require.js?cdv=83:788
callGetModule @ require.js?cdv=83:1203
completeLoad @ require.js?cdv=83:1590
onScriptLoad @ require.js?cdv=83:1717
load (async)
req.load @ require.js?cdv=83:1942
load @ require.js?cdv=83:1685
load @ require.js?cdv=83:834
fetch @ require.js?cdv=83:824
check @ require.js?cdv=83:856
enable @ require.js?cdv=83:1176
enable @ require.js?cdv=83:1557
(anonymous) @ require.js?cdv=83:1161
(anonymous) @ require.js?cdv=83:134
each @ require.js?cdv=83:59
enable @ require.js?cdv=83:1113
init @ require.js?cdv=83:788
(anonymous) @ require.js?cdv=83:1460
setTimeout (async)
req.nextTick @ require.js?cdv=83:1815
localRequire @ require.js?cdv=83:1449
requirejs @ require.js?cdv=83:1797
loadTempl @ util.js?cdv=83:21
loadTemplate @ util.js?cdv=83:64
(anonymous) @ main.js?cdv=83:294
r.complete @ jquery.js?cdv=83:2
c @ jquery.js?cdv=83:2
fireWith @ jquery.js?cdv=83:2
u @ jquery.js?cdv=83:2
S.fx.tick @ jquery.js?cdv=83:2
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
S.fx.start @ jquery.js?cdv=83:2
S.fx.timer @ jquery.js?cdv=83:2
ft @ jquery.js?cdv=83:2
a @ jquery.js?cdv=83:2
dequeue @ jquery.js?cdv=83:2
(anonymous) @ jquery.js?cdv=83:2
setTimeout (async)
(anonymous) @ jquery.js?cdv=83:2
dequeue @ jquery.js?cdv=83:2
(anonymous) @ jquery.js?cdv=83:2
each @ jquery.js?cdv=83:2
each @ jquery.js?cdv=83:2
queue @ jquery.js?cdv=83:2
S.fn.delay @ jquery.js?cdv=83:2
(anonymous) @ main.js?cdv=83:291
r.complete @ jquery.js?cdv=83:2
c @ jquery.js?cdv=83:2
fireWith @ jquery.js?cdv=83:2
u @ jquery.js?cdv=83:2
S.fx.tick @ jquery.js?cdv=83:2
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
S.fx.start @ jquery.js?cdv=83:2
S.fx.timer @ jquery.js?cdv=83:2
ft @ jquery.js?cdv=83:2
a @ jquery.js?cdv=83:2
dequeue @ jquery.js?cdv=83:2
(anonymous) @ jquery.js?cdv=83:2
each @ jquery.js?cdv=83:2
each @ jquery.js?cdv=83:2
queue @ jquery.js?cdv=83:2
animate @ jquery.js?cdv=83:2
handleLoadPanel @ main.js?cdv=83:288
handleClickOnHoverMenuItem @ main.js?cdv=83:918
dispatch @ jquery.js?cdv=83:2
v.handle @ jquery.js?cdv=83:2
main.js?cdv=83:52 Error: Script error for "vs/editor/editor.main"
http://requirejs.org/docs/errors.html#scripterror
    at makeError (require.js?cdv=83:168:17)
    at HTMLScriptElement.onScriptError (require.js?cdv=83:1738:36)
requirejs.onError @ main.js?cdv=83:52
onError @ require.js?cdv=83:549
onScriptError @ require.js?cdv=83:1738
error (async)
req.load @ require.js?cdv=83:1943
load @ require.js?cdv=83:1685
load @ require.js?cdv=83:834
fetch @ require.js?cdv=83:824
check @ require.js?cdv=83:856
enable @ require.js?cdv=83:1176
enable @ require.js?cdv=83:1557
(anonymous) @ require.js?cdv=83:1161
(anonymous) @ require.js?cdv=83:134
each @ require.js?cdv=83:59
enable @ require.js?cdv=83:1113
init @ require.js?cdv=83:788
(anonymous) @ require.js?cdv=83:1460
setTimeout (async)
req.nextTick @ require.js?cdv=83:1815
localRequire @ require.js?cdv=83:1449
requirejs @ require.js?cdv=83:1797
initConfigConsole @ ConfigConsole.js?cdv=83:240
init @ ConfigConsole.js?cdv=83:196
(anonymous) @ util.js?cdv=83:30
execCb @ require.js?cdv=83:1696
check @ require.js?cdv=83:878
(anonymous) @ require.js?cdv=83:1139
(anonymous) @ require.js?cdv=83:134
(anonymous) @ require.js?cdv=83:1189
each @ require.js?cdv=83:59
emit @ require.js?cdv=83:1188
check @ require.js?cdv=83:938
(anonymous) @ require.js?cdv=83:1139
(anonymous) @ require.js?cdv=83:134
(anonymous) @ require.js?cdv=83:1189
each @ require.js?cdv=83:59
emit @ require.js?cdv=83:1188
check @ require.js?cdv=83:938
enable @ require.js?cdv=83:1176
init @ require.js?cdv=83:788
callGetModule @ require.js?cdv=83:1203
completeLoad @ require.js?cdv=83:1590
onScriptLoad @ require.js?cdv=83:1717
load (async)
req.load @ require.js?cdv=83:1942
load @ require.js?cdv=83:1685
load @ require.js?cdv=83:834
fetch @ require.js?cdv=83:824
check @ require.js?cdv=83:856
enable @ require.js?cdv=83:1176
enable @ require.js?cdv=83:1557
(anonymous) @ require.js?cdv=83:1161
(anonymous) @ require.js?cdv=83:134
each @ require.js?cdv=83:59
enable @ require.js?cdv=83:1113
init @ require.js?cdv=83:788
callGetModule @ require.js?cdv=83:1203
completeLoad @ require.js?cdv=83:1590
onScriptLoad @ require.js?cdv=83:1717
load (async)
req.load @ require.js?cdv=83:1942
load @ require.js?cdv=83:1685
load @ require.js?cdv=83:834
fetch @ require.js?cdv=83:824
check @ require.js?cdv=83:856
enable @ require.js?cdv=83:1176
enable @ require.js?cdv=83:1557
(anonymous) @ require.js?cdv=83:1161
(anonymous) @ require.js?cdv=83:134
each @ require.js?cdv=83:59
enable @ require.js?cdv=83:1113
init @ require.js?cdv=83:788
(anonymous) @ require.js?cdv=83:1460
setTimeout (async)
req.nextTick @ require.js?cdv=83:1815
localRequire @ require.js?cdv=83:1449
requirejs @ require.js?cdv=83:1797
loadTempl @ util.js?cdv=83:21
loadTemplate @ util.js?cdv=83:64
(anonymous) @ main.js?cdv=83:294
r.complete @ jquery.js?cdv=83:2
c @ jquery.js?cdv=83:2
fireWith @ jquery.js?cdv=83:2
u @ jquery.js?cdv=83:2
S.fx.tick @ jquery.js?cdv=83:2
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
S.fx.start @ jquery.js?cdv=83:2
S.fx.timer @ jquery.js?cdv=83:2
ft @ jquery.js?cdv=83:2
a @ jquery.js?cdv=83:2
dequeue @ jquery.js?cdv=83:2
(anonymous) @ jquery.js?cdv=83:2
setTimeout (async)
(anonymous) @ jquery.js?cdv=83:2
dequeue @ jquery.js?cdv=83:2
(anonymous) @ jquery.js?cdv=83:2
each @ jquery.js?cdv=83:2
each @ jquery.js?cdv=83:2
queue @ jquery.js?cdv=83:2
S.fn.delay @ jquery.js?cdv=83:2
(anonymous) @ main.js?cdv=83:291
r.complete @ jquery.js?cdv=83:2
c @ jquery.js?cdv=83:2
fireWith @ jquery.js?cdv=83:2
u @ jquery.js?cdv=83:2
S.fx.tick @ jquery.js?cdv=83:2
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
requestAnimationFrame (async)
st @ jquery.js?cdv=83:2
S.fx.start @ jquery.js?cdv=83:2
S.fx.timer @ jquery.js?cdv=83:2
ft @ jquery.js?cdv=83:2
a @ jquery.js?cdv=83:2
dequeue @ jquery.js?cdv=83:2
(anonymous) @ jquery.js?cdv=83:2
each @ jquery.js?cdv=83:2
each @ jquery.js?cdv=83:2
queue @ jquery.js?cdv=83:2
animate @ jquery.js?cdv=83:2
handleLoadPanel @ main.js?cdv=83:288
handleClickOnHoverMenuItem @ main.js?cdv=83:918
dispatch @ jquery.js?cdv=83:2
v.handle @ jquery.js?cdv=83:2
main.js?cdv=83:53 Uncaught Error: Script error for "vs/editor/editor.main"
http://requirejs.org/docs/errors.html#scripterror
    at makeError (require.js?cdv=83:168:17)
    at HTMLScriptElement.onScriptError (require.js?cdv=83:1738:36)
```

## Affected version
* [ ] 09.11.00

## Affected browser
* [ ] Chrome
* [ ] Firefox

**Comment by eXistenZe** [comment]
Probably a duplicate for
https://github.com/dnnsoftware/Dnn.Platform/issues/4538
But I do have a missing file.

**Comment by bdukes** [comment]
We've resolved #5322 via #5328, which fixes this issue when the alias includes something more than the root of the domain (in this case, `/pt-pt`).

You've found `loader.js` in the file system at `Resources\Shared\components\MonacoEditor`, but not `editor\editor.main.js`? Do you have an `editor` folder? If so, what's inside the `editor` folder?

**Comment by bdukes** [comment]
👍🏻closed via #5328
